src/pages/content/actionable-language.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Actionable language
+---
+
+For calls to action (CTAs), try to be as specific as possible.
+This helps the user know exactly what to expect and what they can
+do when they interact with the CTA. Use this guide to find the most
+appropriate verb for an experience that includes CTAs.
+
+CTAs are most often used in [buttons](/components/button) and [anchors](/components/anchor).
+
+## To make a selection
+
+### Select
+
+Use “Select” when the user must make a decision among multiple options.
+
+## To give a person ownership
+
+### Assign
+
+Use “Assign” when referring to ownership and responsibility.
+
+## To move back and forth
+
+### Back
+
+Use “Back” to take the user to the previous step in a multi-step process.
+Pair this with [“Next”](#next). In the event that a user will lose changes,
+warn them ahead of time that they will lose any changes that have been made.
+
+### Previous
+
+Use “Previous” instead of [“Back”](#back) for pagination in tables.
+
+### Next
+
+Use “Next” to take the user to the next step in a multi-step process.
+Pair this with [“Back”](#back).
+
+- **Don’t use:** Proceed, Continue
+
+## To postpone action
+
+### Skip
+
+Use “Skip” to allow the user to postpone an action or decision.
+It’s a way for the user to avoid interruption and proceed with their current task.
+
+## To receive confirmation
+
+### Done
+
+Use “Done” to confirm the completion of a multi-step process.
```

Comment:
Does this conflict with https://garden.zendesk.com/content/word-list#done?

Reply:
Yes it does, thanks for catching @jzempel 
@Francois-Esquire - Same thing here: I'll bring to Chelsea and let you know on how to resolve this!

Reply:
@Francois-Esquire I'm back! And with confirmation from Chelsea on my changes.

I forgot to mention to delete Done from Garden word list when we were jamming together. Sorry! Could you please remove it 🙏